calico/felix/config.py

```diff
@@ -147,7 +147,7 @@ def __init__(self, config_path):
         """
         Create a config. This reads data from the following sources.
         - Environment variables
-        - Configuration file
+        - Configuration file - (/etc/calico/felix.cfg)
```

Comment:
Nitpick: for consistency with the lines below, probably don't want a dash and a paren `- (`.